src/scrabbag.js

```diff
@@ -0,0 +1,50 @@
+//input a string
+//split string into an array
+//iterate over the array
+//for each letter and subtract one
+//MAKE THE STRING UPPERCASE
+
+var scrabbleBag = {
```

Comment:
Best practice is to use `let` or `const` to avoid the possibility of subtle declaration errors.  This should be consistently applied in the rest of this code, as well.